cxf-stub/ec2-2014-02-01.wsdl

```diff
@@ -1443,6 +1443,7 @@
       <xs:complexType name="DescribeVolumesResponseType">
         <xs:sequence>
           <xs:element name="requestId" type="xs:string"/>
+          <xs:element name="nextToken" type="xs:string" minOccurs="0"/>
```

Comment:
This field has been added in recent AWS API, right?
If you can find a lately full version of new ec2-xxx.wsdl with such new fields, that would be great!